cmd/hatchet-loadtest/load_e2e_test.go

```diff
@@ -37,47 +30,133 @@ func TestLoadCLI(t *testing.T) {
 
 	tests := []struct {
 		name    string
-		args    args
+		config  LoadTestConfig
 		wantErr bool
 	}{
 		{
 			name: "test with high step delay",
-			args: args{
-				duration:        240 * time.Second,
-				eventsPerSecond: 10,
-				delay:           10 * time.Second,
-				wait:            60 * time.Second,
-				concurrency:     0,
+			config: LoadTestConfig{
+				Duration:       240 * time.Second,
+				Events:         10,
+				Delay:          10 * time.Second,
+				Wait:           60 * time.Second,
+				Concurrency:    0,
+				Slots:          100,
+				FailureRate:    0.0,
+				PayloadSize:    "0kb",
+				EventFanout:    1,
+				DagSteps:       1,
+				RlKeys:         0,
+				RlLimit:        0,
+				RlDurationUnit: "",
 			},
 		}, {
 			name: "test simple with unlimited concurrency",
-			args: args{
-				duration:        240 * time.Second,
-				eventsPerSecond: 10,
-				delay:           0 * time.Second,
-				wait:            60 * time.Second,
-				concurrency:     0,
+			config: LoadTestConfig{
+				Duration:       240 * time.Second,
+				Events:         10,
+				Delay:          0 * time.Second,
+				Wait:           60 * time.Second,
+				Concurrency:    0,
+				Slots:          100,
+				FailureRate:    0.0,
+				PayloadSize:    "0kb",
+				EventFanout:    1,
+				DagSteps:       1,
+				RlKeys:         0,
+				RlLimit:        0,
+				RlDurationUnit: "",
+			},
```

Comment:
this looks similar to the one above to me - what am I missing here? I think the only difference is the `Delay`?

Reply:
yeah this should be a bit more cleaned up I think